### DISCUSSION TITLE
What is the colorscheme in the screenshot?

### DISCUSSION THREAD

**Discussion by samrathchadha** [MunifTanjim/nui.nvim#39] [Q&A] [answered]

**Comment by MunifTanjim** [comment]
Hey @samrath2007, I don't know for sure, but I think it's a custom made colorscheme: https://github.com/vuki656/nvim-config/blob/89ba84f7570aece799cac823ca92a644748a1669/lua/colorscheme.lua

@vuki656 made those screenshots, so he would know better 😃

**Comment by vuki656** [comment, marked as answer]
Hey, it's https://github.com/rakr/vim-one with some modifications [here](https://github.com/vuki656/nvim-config/blob/master/lua/colorscheme.lua)

**Comment by samrathchadha** [comment]
Thank you!
